src/vanilla-tilt.js

```diff
@@ -202,3 +202,17 @@ if (typeof document !== "undefined") {
    */
   VanillaTilt.init(document.querySelectorAll("[data-tilt]"));
 }
+
+/**
+ * Module Wrapper
+ * --------------
+ */
+if (typeof define === 'function' && typeof define.amd === 'object' && define.amd) {
+  define(function () {
+    return VanillaTilt
+  })
+} else if (typeof module !== 'undefined' && module.exports) {
+  module.exports = VanillaTilt
```

Comment:
@micku7zu I couldn't write this in ES5 because babel doesn't know how to compile this.